Lesson_10_Algorithm/Program.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+
+class Program
+{
+    static void Main(string[] args)
+    {
+        int[] array = { 4, 9, 7, 6, 2, 3, 8, 5, 0, 1 };
+
+        SortMathod(array);
+
+        PrintArray(array);
+    }
+
+    private static void SortMathod(int[] array)
+    {
+        for (int i = 0; i < array.Length - 1; i++)
+        {
+            for (int j = i + 1; j < array.Length; j++)
+            {
+                if (array[i] > array[j])
+                {
+                    int temp = array[i];
```

Comment:
Try to swap values without the additional variable. It isn't so useful, but it is a challenging task for breaking your brain. This kind of task can train your brain and this is beneficial for you.